tools/cbmc/proofs/make_configuration_directories.py

```diff
@@ -125,13 +125,18 @@ def process(folder, files):
                 """))
             LOGGER.error("The offending entry is %s", config)
             return
+
+        new_entry = "%s__config_%s" % (json_content["ENTRY"], configname)
```

Comment:
The problem is clashing batch job names resulting in clashing S3 folder names.  Don't the job names come from the name of the directory holding the proof files?  Don't we need to change the directory name to "proofname_config_configname" and not just the harness name?

Reply:
Done (see line 131).